### ISSUE TITLE
Folder sorting does not seem to work in compact display mode

### ISSUE THREAD

**Issue by jjhc123** [protz/Manually-Sort-Folders#187]
Thunderbird 91.10.0.

I've reordered my accounts and this works well. I usually use 'Favorite Folders' and everything is fine.

Sadly, when I turn on 'Compact view' (on the Folders toolbar) my manual ordering gets ignored. My favorite folders suddenly get reordered to alphabetical.

Am I missing something terribly obvious here? Or is something just not quite working right?

Thanks for any assistance.

**Comment by itagagaki** [comment]
I will look into it.

**Comment by itagagaki** [comment]
This seems to be a problem with Thunderbird itself.
It was reported as a bug 3 years ago, but it still seems to be left unassigned to anyone.
https://bugzilla.mozilla.org/show_bug.cgi?id=1606116

**Comment by jjhc123** [comment]
Thank you for investigating this. It's annoying, but I can put up with it. I do like the functionality of the basic extension, so I thank you for this. You can close this issue if you desire.

**Comment by itagagaki** [comment]
I have created and submitted a patch to Thunderbird that resolves this issue. The patch is currently waiting on review. Let's wait for good news.

**Comment by itagagaki** [comment]
Good news, the fix for this issue that I submitted to Mozilla has been accepted and merged into the Thunderbird repository.
The planned version is 109, so it will take some more time.
